mkosi

```diff
@@ -4720,7 +4692,7 @@ def print_summary(args: CommandLineArguments) -> None:
         sys.stderr.write("          Repositories: " + ','.join(args.repositories) + "\n")
     sys.stderr.write("\nOUTPUT:\n")
     if args.hostname:
-        sys.stderr.write("              Hostname: " + args.hostname + "\n")
+        sys.stderr.write("                  Hostname: " + args.hostname + "\n")
```

Comment:
This whitespace moves the colon out of the line, doesn't it?

Reply:
Ugh my bad I had to indent these back again after the option name changed and it seems I forgot some.